packages/client/tests/e2e/browser-enum/_steps.ts

```diff
@@ -0,0 +1,17 @@
+import { $ } from 'zx'
+
+import { executeSteps } from '../_utils/executeSteps'
+
+void executeSteps({
+  setup: async () => {
+    await $`pnpm install`
+    await $`pnpm exec prisma generate`
+  },
+  test: async () => {
+    await $`pnpm exec jest`
```

Comment:
I think we could also typecheck?

Reply:
We don't actually have `.d.ts` file for `index-browser`, so `tsc` fails with direct import. I think better test for this would be importing `@prisma/client`,  bundling it and running it in proper browser-like environment (jsdom? pupeeteer?), but for the sake of getting the fix faster, i'd suggest we do afterwards.

Reply:
:+1: